docs/ribs.docs

```diff
@@ -99,6 +99,51 @@ and given that `time` was the "time since the program started", this
 would draw a sprite at (100, 100) shrunk to half size along the y axis, rotating
 around it's center.
 
+# `draw_text(text, position, size=32, color=pg.Color(255, 255, 255), font=None)`
+Draw `text` at `position`, which is given in pixels from the top left corner.
+Optional arguments include `size` given in pixels, `color` which is
+a standard pygame color and `font` which is a string. To get a list of
+available fonts, use `pg.font.get_fonts()`.
+!--params
+[text] The text to be drawn.
+[position] The top left corner of the text, in pixel coordinates.
+[size] The font size in pixels. (Optional)
```

Comment:
Have you double checked with the PyGame docs that it's pixels? I think they often use points, which is not the same as pixels... When I read the docs the size looked to be in points.

Reply:
This is probably true, i will change it.